src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -210,7 +210,7 @@ void Player_Action_97(Player* this, PlayState* play);
 
 s32 Player_UpperAction_0(Player* this, PlayState* play);
 s32 Player_UpperAction_1(Player* this, PlayState* play);
-s32 Player_UpperAction_2(Player* this, PlayState* play);
+s32 Player_UpperAction_ChangeHeldItem(Player* this, PlayState* play);
```

Comment:
Worth noting that in OoT, these functions are called `_IA_` or "ItemAction" functions, which seems to be the use case for most of these, but not in all cases, such as carrying an actor above your head. UpperAction is still an option to name these, so I left it alone for now. But it can always change in a later PR

Reply:
I am heavily leaning toward changing oots to upper action as well. Some desync between games here is fine in the interim until that all gets sorted